tools/gen_compilation_database.py

```diff
@@ -76,12 +92,13 @@ def is_compile_target(target, args):
     return True
 
 
-def modify_compile_command(target, args):
+def modify_compile_command(target, args, output_base):
     cc, options = target["command"].split(" ", 1)
 
-    # Strip wrapper path so that clangd can find the compiler.
+    # cc_wrapper.sh is a script of llvm_toolchain to wrap clang/clang++. Make sure to
+    # use the one from output_base.
     if cc.endswith("cc_wrapper.sh"):
-        cc = "cc_wrapper.sh"
+        cc = os.path.join(output_base, cc)
```

Comment:
Get the correct binary full path so we can avoid the dependency to the soft link. This make sure this script could be useful even the dev container is not used.